data/data/powervs/cluster/bootstrap/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable "memory" {
+  type = string
+  description = "The amount of memory to assign to each node in GB."
+}
+
+variable "processors" {
+  type = string
+  description = "The processor count for each node."
+}
+
+variable "ignition" {
+  type = string
+  description = "The ignition file."
```

Comment:
I think this is okay, but "The contents of the ignition file" may be more accurate.